aodncore/pipeline/templates/notify.html.j2

```diff
@@ -42,8 +42,20 @@
     </div>
     {% endif %}
 
+    <p>
+    For further information about this message, please contact <a href="mailto:info@aodn.org.au">info@aodn.org.au</a>
+    (or the relevant AODN project officer).
```

Comment:
The "relevant project officer" bit makes me think there should be a per-pipeline list of POs in the notify params that could be used here? Maybe a future enhancement though...

Reply:
Yep, another possible use for a pipeline "owners" list (https://github.com/aodn/zzz-aodn-pipeline-poc/issues/162), but I think the info@aodn will be enough for now.